draft-ietf-quic-transport.md

```diff
@@ -2432,17 +2432,18 @@ The capacity available on the new path might not be the same as the old path.
 Packets sent on the old path MUST NOT contribute to congestion control or RTT
 estimation for the new path.
 
-On confirming a peer's ownership of its new address, an endpoint MUST
+On confirming a peer's ownership of its new address, an endpoint SHOULD
```

Comment:
Why was this changed to a MUST? I thought if there is "unless" condition, MUST is the right choice.

Reply:
It was changed to SHOULD because we added the "unless" clause. This is exactly what a SHOULD is for?

Reply:
This is a thing I'm never sure about but I would use MUST with an explicit exception. While SHOULD would mean there might be more exceptions...

Reply:
I was under the impression that a MUST with an exception was a SHOULD, per 2119. I have no problems if that understanding is wrong, although a MUST...unless just feels odd. What do others think?

Reply:
We've used "MUST ... unless" in the document to carve out specific exceptions. RFC 2119 says that violating a SHOULD is allowed if the implementer knows what they're doing. "MUST ... unless" is more prescriptive -- it says that the implementer can ignore the MUST only under the exceptional condition.

In this instance, I agree with @mirjak that "MUST ... unless" is the intent.